test/system/090-events.bats

```diff
@@ -61,3 +61,34 @@ load helpers
 .*image remove $imageID $tag.*" \
        "podman events"
 }
+
+function _events_disjunctive_filters() {
+    local backend=$1
+
+    # Regression test for #10507: make sure that filters with the same key are
+    # applied in disjunction.
+    t0=$(date --iso-8601=seconds)
+    run_podman $backend run --name foo --rm $IMAGE ls
+    run_podman $backend run --name bar --rm $IMAGE ls
+    run_podman $backend events --stream=false --since=$t0 --filter container=foo --filter container=bar --filter event=start
+    is "$output" ".* container start .* name=foo.*
+.* container start .* name=bar.*"
+}
+
+@test "events with disjunctive filters - file" {
+    skip_if_remote "remote does not support --events-backend"
+    _events_disjunctive_filters --events-backend=file
+}
+
+@test "events with disjunctive filters - journald" {
+    skip_if_remote "remote does not support --events-backend"
+    _events_disjunctive_filters --events-backend=journald
+}
+
+@test "events with disjunctive filters - remote only" {
+    skip "FIXME: remote events seems to eat events"
+    # Since podman-remote does not support the --events-backend flag, do a
+    # remote-only test with the default events backend.
+    skip_if_local "test is intended to only run on remote"
```

Comment:
Why the skip? What harm does it do to test the default events backend when running local? It actually seems like a kind of important thing to test.

Reply:
>The default is implicitly tested by the two tests above

Quick clarification: *Today's* default is *explicitly* tested. Tomorrow, someone may change the default to `--events-backend=something-new`. Or, the default might actually not be implemented correctly, or it might break in the future. Paranoia is a good thing sometimes: it would be really embarrassing to have the default case break, and ship!

Reply:
> Or, the default might actually not be implemented correctly, or it might break in the future. Paranoia is a good thing sometimes: it would be really embarrassing to have the default case break, and ship!

Sounds good. Changed :+1: